deepspeed/pt/deepspeed_zero_optimizer.py

```diff
@@ -95,10 +96,45 @@ def move_to_cpu(tensor_list):
         tensor.data = tensor.data.cpu()
 
 
+def move_to_cuda(tensor_list, cuda_device):
+    for tensor in tensor_list:
+        tensor.data = tensor.data.to(cuda_device)
+
+
 def print_rank_msg(msg):
     print(f"rank {dist.get_rank()} - {msg}")
 
 
+def async_migrate_to(obj, dev, main_stream=None):

```

Comment:
Can we have standalone unit tests for `async_migrate_to()` and `async_copy_to()`?

Reply:
This is TODO for a follow on PR.